src/engine/ruleset/decoders/sophos/utm/sophos-utm.yml

```diff
@@ -0,0 +1,188 @@
+---
+name: decoder/sophos-utm/0
+
+metadata:
+  module: Sophos
+  title: Sophos-Utm logs decoder
+  version: [17.5.x, 18.0.x]
+  description: Decoder for Sophos Utm
+  author:
+    name: Wazuh Inc. info@wazuh.com
+    date: 2023-01-12
+  references:
+    - https://www.sophos.com/en-us/support/documentation/sophos-utm
+
+sources:
+  - decoder/queue-syslog/0
+  - decoder/queue-localfile/0
+
+parse:
+  logpar:
+
+    # 2016:1:29-06:09:59 localhost.localdomain smtpd[905]: MASTER[nnumqua]: QR globally disabled, status one set to 'disabled'
+    - event.original: <@timestamp/%Y:%m:%d-%H:%M:%S> <host.hostname> <event.code>[<~tmp.process_pid>]:<~/ignore/ ><~>[<~>]:<~/ignore/ ><~tmp.message>
+
+    # 2017:3:18-18:24:33 hostname reverseproxy: [tutla] [isund:high] [pid 3136:uidex] [client uptate] Invalid signature, cookie: JSESSIONID
+    # 2016:3:26-10:20:16 ctetura3009.www5.corp reverseproxy: [lita] [adeseru:medium] [pid 7692:eaq] amest configured -- corp normal operations
+    - event.original: <@timestamp/%Y:%m:%d-%H:%M:%S> <host.hostname> <event.code>:<~/ignore/ >[<~>] [<~tmp.eventlog>:<~tmp.severity>] [pid <~tmp.process_pid>:<~>] <~tmp.message>
```

Comment:
This log complies with RFC 3164, it should not be a sibling of the syslog decoder log.